src/oc_erchef/schema/revert/create_and_update_users.sql

```diff
@@ -0,0 +1,8 @@
+-- Revert enterprise_chef:create_and_update_users from pg
+
+BEGIN;
+
+DROP FUNCTION IF EXISTS add_user();
+DROP FUNCTION IF EXISTS update_user();
```

Comment:
The deploy script implies that these functions existed in the system before, but this revert will not replace those older implementations.

Reply:
Yes - this is just safeguarding for the future, we don't expect the functions to actually be present.